.github/workflows/build_library.yml

```diff
@@ -43,15 +43,15 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - macos-version: "10.15"
+          - macos-version: "macos-10.15"
             target: x86_64-apple-darwin
             py-platform: macosx-10_15_x86_64
-          - macos-version: "11.0"
+          - macos-version: "macos-11.0"
             target: aarch64-apple-darwin
             py-platform: macosx-11_0_arm64
 
     name: Python macOS ${{ matrix.py-platform }}
-    runs-on: macos-${{ matrix.macos-version }}
+    runs-on: ${{ matrix.macos-version }}
```

Comment:
this needed a slight update since the schema doesn't understand partially interpolated values: https://github.com/python-jsonschema/check-jsonschema/issues/110